src/Misc/ConsoleResponse.php

```diff
@@ -114,6 +114,9 @@ public function getResponseBody(): string
     public function getFormattedJsonBody(): string
     {
         $body = $this->responseBody;
+        if ($body === null){
+            $body = '{}';
```

Comment:
I think we can simply return empty string here, or not?

Reply:
Yes, you're right. I changed that